package.json

```diff
@@ -4,17 +4,26 @@
   "description": "A module for getting method signature info from an ethereum method signature.",
   "main": "index.js",
   "scripts": {
-    "lint": "echo 'No lint script'",
+    "lint": "eslint . --ext js,json",
+    "lint:fix": "yarn lint --fix",
     "test": "node test"
   },
   "author": "Dan Finlay <dan@danfinlay.com>",
   "license": "ISC",
   "dependencies": {
-    "ethjs": "^0.3.0"
+    "ethjs": "^0.3.0",
+    "typescript": "^4.1.3"
```

Comment:
TypeScript should be moved to `devDependencies`. You can just manually insert it in the right place, alphabetically.